### PR TITLE
Add text fields for `custom_modules` and `extra_suffix`

### PR DESCRIPTION
**Pull request by greenfox1505**

### CODE REVIEW

**Comment by greenfox1505** [comment]
Sorry, I think my auto formatter got a little bit over zealous. I can stage just the lines with relevant changes if that matters.

File: index.html [hunk 1]
```diff
@@ -490,4 +516,5 @@ <h2>Resulting <code>custom.py</code></h2>
     }
   </script>
 </body>
-</html>
+
+</html>
```
Comment (Calinou) [inline index.html]:
```suggestion
</html>

```

File: index.html [hunk 2]
```diff
@@ -70,6 +75,14 @@ <h2>Resulting <code>custom.py</code></h2>
     function buildOptionsGenerator() {
       return {
         options: [
+          {
+            id: "extra_suffix",
+            name: "Extra Suffix",
+            description: 'Changed the "custom_build" suffix. Useful for adding your "game_title" to a build',
```
Comment (Calinou) [inline index.html]:
```suggestion
            description: 'Changes the "custom_build" suffix. Useful for adding your "game_title" to a build.',
```

File: index.html [hunk 3]
```diff
@@ -443,6 +456,14 @@ <h2>Resulting <code>custom.py</code></h2>
             enabled: true,
             default: true,
           },
+          {
+            id: "custom_modules",
+            name: "Custom Module Path",
+            description: 'Addes a path for your custom models. Useful for keeping custom modules seperate from the Godot repo.',
```
Comment (Calinou) [inline index.html]:
```suggestion
            description: 'Specifies a path to a folder containing custom C++ modules to build. Useful for keeping custom modules separate from the Godot repository.',
```

File: index.html [hunk 4]
```diff
@@ -70,6 +75,14 @@ <h2>Resulting <code>custom.py</code></h2>
     function buildOptionsGenerator() {
       return {
         options: [
+          {
+            id: "extra_suffix",
+            name: "Extra Suffix",
```
Comment (Calinou) [inline index.html]:
```suggestion
            name: "Extra suffix",
```

File: index.html [hunk 5]
```diff
@@ -443,6 +456,14 @@ <h2>Resulting <code>custom.py</code></h2>
             enabled: true,
             default: true,
           },
+          {
+            id: "custom_modules",
+            name: "Custom Module Path",
```
Comment (Calinou) [inline index.html]:
```suggestion
            name: "Custom modules path",
```

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Pull request merged**